sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -398,6 +400,13 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class DelayStrategy
+    {
+        protected DelayStrategy() { }
+        public abstract System.TimeSpan GetNextDelay(Azure.Response? response, int attempt, System.TimeSpan? delayHint);
```

Comment:
@KrzysztofCwalina's idea: clientDelayHint and serverDelayHint params. clientDelayHint would be based on user specified delay/retry policy calculation. serverDelayHint would be based on retry-after header. Since users can always call Update themselves in a loop for LROs, they can always choose to bypass the retry-after header, so it may be sufficient to just present them with the value and let them decide what to do.

Reply:
@m-nash thoughts on the above?